.github/workflows/build.yml

```diff
@@ -29,25 +29,18 @@ jobs:
         sudo apt-get update
         sudo apt-get install \
             ccache \
-            libgtk2.0-0 \
-            libncursesw5 \
-            libsdl-image1.2-dev \
-            libsdl-ttf2.0-dev \
-            libsdl1.2-dev \
-            libxml-libxml-perl \
             libxml-libxslt-perl \
-            lua5.3 \
-            ninja-build \
-            zlib1g-dev
+            libgl-dev \
```

Comment:
not (quite) sorted

Reply:
I didn't even notice that there was sorting going on : p will fix

Reply:
done